database/queryHelpers.js

```diff
@@ -1,4 +1,40 @@
 const { client } = require('./client');
+const { elasticClient } = require('./elasticsearch.js');
+
+// ELASTIC QUERIES
+
+module.exports.getAveragePriceElastic = (parameters, startDate, endDate) => {
+// console.log(parameters, startDate, endDate);
+  const queryArray = Object.keys(parameters).map((key) => {
+    const obj = {};
+    obj[`doc.${key}`] = parameters[key];
+    return { match: obj };
+  });
+
+  return elasticClient.search({
+    index: 'bookings',
+    type: 'booking',
+    size: 10000,
+    body: {
+      query: {
+        bool: {
+          must: queryArray.concat([
+            {
+              range: {
+                'doc.booked_at': { gte: startDate, lte: endDate },
+              },
+            },
+          ]),
+        },
+      },
+//      sort: { date: 'desc' },
```

Comment:
I'm actually going to leave this in, because currently this is not functional because I am not getting real flowing data from Inventory service yet. When I do get it, this will be required to sort the date by.